app/jobs/update_job.rb

```diff
@@ -15,7 +17,14 @@ class UpdateJob < ApplicationJob
   # @param [String] version_description
   # rubocop:disable Metrics/AbcSize
   # rubocop:disable Metrics/MethodLength
-  def perform(model_params:, signed_ids:, background_job_result:, start_workflow: true, version_description: nil)
+  def perform(**kwargs)
```

Comment:
Can you say more about changing `UpdateJob#perform` from keyword arguments to a hash? Is it Rubocop-related? I'm curious because `IngestJob#perform` is still using keyword arguments and these jobs are siblings of sorts.

Reply:
Yes, this was to appease rubocop. I'm OK going back and putting an inline exception, but thought this was more idiomatic.

Reply:
The fact that so many methods in sdr-api use more than 4-5 keyword args is definitely a smell, IMO! I think in this case, I may be slightly biased in favor of internal consistency within the codebase until we find abstractions to cut down on passing around so much knowledge.